packages/genesys-spark-components/src/components/stable/gux-copy-to-clipboard/gux-copy-to-clipboard.tsx

```diff
@@ -32,14 +31,6 @@ export class GuxCopyToClipboard {
     this.resetTooltip();
   }
 
-  @Listen('focus')
-  onFocus() {
-    // when copy button is focused by keyboard it will have `:focus-visible` pseudo class
-    if (this.copyButton.matches(':focus-visible')) {
-      this.tooltipContent = 'enterToCopy';
```

Comment:
I removed this as I thought it was confusing and redundant. Users do not need help to use a basic button via a keyboard.